utils/python/CIME/XML/generic_xml.py

```diff
@@ -37,6 +37,7 @@ def __init__(self, infile=None):
         else:
             # if file does not exist create a root xml element
             # and set it's id to file
+            logger.warning("File %s does not exists." , infile)
```

Comment:
I don't think that we want a warning here - creating a new file will be a regular operation in this init.